mule-extensions-api/src/main/java/org/mule/extension/api/annotation/metadata/MetadataKeyParam.java

```diff
@@ -26,7 +28,7 @@
  *
  * @since 1.0
  */
-@Target(PARAMETER)
+@Target({PARAMETER, FIELD})
```

Comment:
add a paragraph on the javadoc explaining in which cases it is to be used on a parameter and when on a field. Also correct a typo on the javadoc: or or source's -> or source's